.misc/deploy.pypi.sh

```diff
@@ -4,18 +4,22 @@
 set -e
 
 # Get environment variables, readily decrypted by rultor
-source ../rultor_secrets.sh
+#source ../rultor_secrets.sh
 
 # Ship it!
 echo "Uploading coala to pypi"
 pip3 install twine wheel
 python3 setup.py sdist bdist_wheel
 # Upload one by one to avoid timeout
-# Don't block merging because pypi is in error 500 mood again
-set +e
-twine upload dist/*.whl -u "$PYPIUSER" -p "$PYPIPW"
-twine upload dist/*.tar.gz -u "$PYPIUSER" -p "$PYPIPW"
-set -e
+twine upload dist/* -u "$PYPIUSER" -p "$PYPIPW" 2>&1 | tee twine_output.txt
+if [ "${PIPESTATUS[0]}" -ne 0 ]; then
+    grep -q 500 twine_output.txt
+    if [ "$?" -eq 0 ]; then
```

Comment:
are quotes necessary? I know safe is safe in bash, though not sure this is required in this case